README.md

```diff
@@ -5,30 +5,24 @@
 
 This project is an experiment to implement a
 [Language Server](https://github.com/Microsoft/language-server-protocol) for
-Scala using [Scalameta](http://scalameta.org/)-related technologies.
+Scala using [Scalameta](http://scalameta.org/) projects such as
+[Scalafmt](http://scalameta.org/scalafmt/),
+[Scalafix](https://scalacenter.github.io/scalafix/) and
+[SemanticDB](https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md).
 
 :warning: This project is under development and is not intended to be used for
 day-to-day coding. Expect bugs and incomplete documentation. Installation
 instructions are primarily intended for project contributors.
 
-## Project Goals
-
-* simple installation
-* correct diagnostics (red squiggles)
-* robust navigation (goto definition anywhere)
-* fast completions
-* low CPU and memory usage
-* refactoring (rename, organize imports, move class, insert type annotation)
-
 ## Contributing
 
-See the [installation instructions](BETA.md) and
+See the [installation instructions](docs/installation.md) and
 [contributing guide](CONTRIBUTING.md).
 
 ## Roadmap
 
-See the [roadmap](roadmap.md) for a rough summary of what has been implemented
-so far.
+See the [roadmap](docs/roadmap.md) for an overview of what features have been
+implemented.
```

Comment:
"...and what features we're planning to work on next"?

Reply:
Done.